src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -1110,6 +1110,24 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Security\SecureString.Unix.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\TimeZoneInfo.Unix.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(FeaturePortableThreadPool)' == 'true'">
+    <Compile Include="$(MSBuildThisFileDirectory)System\Threading\ThreadPool.Portable.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Threading\ClrThreadPool.cs" />
```

Comment:
Little bit confusing naming here, is ClrThreadPool equal to ThreadPool.Portable ?

Reply:
I kept the original naming of the files. There's `ThreadPool.Portable.cs` which implements the `ThreadPool` methods. The other files are the internal implementation classes, which are created and called from the the `ThreadPool` code.

Reply:
I'd prefer to follow the existing naming convention. ThreadPool.cs for portable code, ThreadPool.CoreCLR.cs for CoreCLR runtime specific implementation

Reply:
There is already `ThreadPool.cs` shared between all implementations. Then there are three implementations of the actual thread pool (CoreCLR, CortRT/Portable, CoreRT/Windows).

Reply:
> If it helps, it would be ok to create PortableThreadPool subdirectory and move the portable threadpool implementation there.

And rename ClrThreadPool.*.cs to PortableThreadPool.*.cs, presumably, or something along those lines.